app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue

```diff
@@ -174,17 +174,25 @@ export default {
     onSuccess() {
       this.$emit('success');
     },
-
+    getConversationAlert(conversationData) {
+      return `${this.$t('NEW_CONVERSATION.FORM.SUCCESS_MESSAGE')}
+           </br> <a href="/app/accounts/${
+             conversationData.account_id
+           }/conversations/${conversationData.id}">${this.$t(
+        'NEW_CONVERSATION.FORM.GO_TO_CONVERSATION'
+      )}</a>
```

Comment:
Passing an HTML seems a little off, can use additional params on the SnackBar component. Payload can be of the format.
```
{
   message: '',
   action: {
     type: 'link',
     to: '<link-to-url>'
   }
}
```